src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
I would also be down for using functions. I've always felt nervous about using `node` / `nodes` as I felt like it was reaching into enzyme internals that could be changed without notice.. Functions feel more like a public API.